libs/nillion-chain/client/Cargo.toml

```diff
@@ -19,3 +19,6 @@ uuid = "1"
 rand = "0.8.5"
 
 nillion-chain-transactions = { path = "../transactions" }
+
+[dev-dependencies]
+rstest = "0.24.0"
```

Comment:
Can you use rstest 0.21.0 so we don't pull in both? see cargo.lock. We need to bump this all at once to avoid pulling in duplicates, this has happened before.